requirements.txt

```diff
@@ -0,0 +1 @@
+earthdata-varinfo==1.0.1
```

Comment:
Just to check - things continued to run after making this change? (I think they should, but just want to sanity check without setting up a whole new environment - lazy I know!)

Reply:
Yup works with just that dependency.